Mergin/create_project_wizard.py

```diff
@@ -360,14 +362,14 @@ def node_shown(self, node):
 
     def flags(self, idx):
         if idx.column() == self.LAYER_COL:
-            return Qt.ItemIsEnabled
+            return Qt.ItemFlags.ItemIsEnabled
```

Comment:
The Qt.ItemFlag enum should be used instead (without the s) otherwise it crashes when openning the creation wizard
```suggestion
            return Qt.ItemFlag.ItemIsEnabled
```